packages/vscode-extension/package.json

```diff
@@ -693,8 +702,8 @@
       {
         "command": "fx-extension.manageCollaborator",
         "title": "%teamstoolkit.commands.manageCollaborator.title%",
-        "enablement": "fx-extension.isV3Enabled && !fx-extension.commandLocked",
```

Comment:
If the lock condition is removed, will it work when user is performing long-running task like provision?

Reply:
It should not work. Updated, thx!